R/misc.R

```diff
@@ -659,6 +659,24 @@ check_training_set <- function(x, rec, fresh) {
   x
 }
 
+check_new_data_columns <- function(object, new_data) {
```

Comment:
We should have an argument here for role exclusions. We don't want to error if the outcome columns are missing. 

We'll need some time to think about this in regards to what to do about other roles such as case weights, id variables, performance variables and so on.